packages/material-ui/src/MenuList/MenuList.js

```diff
@@ -35,9 +36,6 @@ function textCriteriaMatches(nextFocus, textCriteria) {
     // jsdom doesn't support innerText
     text = nextFocus.textContent;
   }
-  if (text === undefined) {
```

Comment:
Never hit: https://codecov.io/gh/mui-org/material-ui/src/master/packages/material-ui/src/MenuList/MenuList.js#L38

Could not find info why this would be `undefined`. [It can be `null`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) apparently but I wouldn't know when and so far we never encountered this.